src/components/Todo.js

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import * as actions from '../redux/actions';
+
+const Todo = ({ todo, deleteTodo, completeTodo, changeTitle }) => {
+  const [newTitle, setNewTitle] = useState(todo.title);
+  const [editing, setEditing] = useState(false);
+
+  useEffect(() => {
+    document.getElementById(`${todo.id}-edit`).focus();
+  }, [editing, todo]);
+
+  return (
+    <li
+      key={`${todo.id}-key`}
+      className={
+        `${todo.completed
+          ? 'completed'
+          : ''
+        } ${editing
+          ? 'editing'
+          : ''
+        }`
+      }
+    >
+      <div className="view">
+        <input
+          checked={todo.completed}
+          type="checkbox"
+          className={`toggle ${todo.completed ? 'completed' : ''}`}
+          value={todo.completed}
+          onChange={
+            (event) => {
+              completeTodo(todo.id, !todo.completed);
```

Comment:
```suggestion
              completeTodo(todo.id);
```
Передавай только id, остальное делай в методе completeTodo